DigitalLearningSolutions.Web.Tests/Controllers/Support/ResourcesControllerTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.Support
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.Support;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using FakeItEasy;
+    using FluentAssertions.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.FeatureManagement;
+    using NUnit.Framework;
+
+    public class ResourcesControllerTests
+    {
+        private IFeatureManager featureManager = null!;
+        private IConfiguration configuration = null!;
+        private IResourcesService resourcesService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            featureManager = A.Fake<IFeatureManager>();
+            configuration = A.Fake<IConfiguration>();
+            resourcesService = A.Fake<IResourcesService>();
+            A.CallTo(() => featureManager.IsEnabledAsync(FeatureFlags.RefactoredTrackingSystem))
+                .Returns(true);
+        }
+
+        [Test]
+        public async Task Frameworks_Resources_page_should_be_shown_for_valid_claims()

```

Comment:
Should we have a similar test for the tracking system?

Reply:
If this were getting released in the meantime I'd say it is worth it, but it isn't, so fine to leave.